features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/TimelineState.kt

```diff
@@ -35,3 +36,8 @@ data class TimelineState(
     val sessionState: SessionState,
     val eventSink: (TimelineEvents) -> Unit
 )
+
+@Immutable
+data class TimelineRoomInfo(
+    val isDirect: Boolean,
```

Comment:
For now there is just one member, but I prefer to create a data class to be able to add more members later without touching all the code again.